front-page.php

```diff
@@ -0,0 +1,293 @@
+<?php
+get_header();
+?>
+
+<!-- main content -->
+
+<main id="main-content" class="container">
+
+
+<?php 
+  // WP_Query arguments
+  $args = array (
+    'category_name'          => 'featured',
+    'posts_per_page'         => '1',
+  );
+
+  // The Query
+  $featured_query = new WP_Query( $args );
+
+
+  // The Loop
+  if ( $featured_query->have_posts() ) { 
+?>
+
+  <section id="featured-post">
+
+<?php
+    while ( $featured_query->have_posts() ) {
+      $featured_query->the_post();
+
+      $featured_id = $post->ID;
+
+      $subtitle = get_post_meta( $post->ID, '_igv_post_subtitle', true );
+?>
+
+    <article <?php post_class('col s24'); ?> id="post-<?php the_ID(); ?>">
+
+      <a href="<?php the_permalink() ?>">
+
+        <h3 class="featured-title">
+          <?php the_title(); ?>
+        </h3>
+
+        <?php if ($subtitle) { ?>
+        <div class="featured-subtitle">
+          <?php echo $subtitle; ?>
+        </div>
+        <?php } ?>
+
+        <?php the_post_thumbnail(); ?>
+
+      </a>
+
+    </article>
+
+<?php 
+    }
+?>
+
+  </section>
+
+<?php 
+  } 
+  wp_reset_postdata(); 
+?>
+
+
+  <section class="row"> <!-- Puta portadazza, Noticias, Ads -->
+
+    <div class="col s8">
+
+<?php 
+  // WP_Query arguments
+  $args = array (
+    'category_name'          => 'puta-portadazza',
+    'posts_per_page'         => '1',
+  );
+
+  // The Query
+  $puta_query = new WP_Query( $args );
+
+  $category = get_category_by_slug( 'puta-portadazza' );
+  $cat_name = $category->cat_name;
+  $cat_id = $category->term_id;
+  $cat_link = get_category_link( $cat_id );
+
+  // The Loop
+  if ( $puta_query->have_posts() ) { 
+?>
+
+      <div class="feed-category">
+        <a href="<?php echo esc_url( $cat_link ); ?>">
```

Comment:
I think these links either need to be in a span with a class or in a header [probably also with a class]. I imagine that the div will have to have explicit width and then the link/text be rotated